fpga_interchange/arch.cc

```diff
@@ -666,8 +712,61 @@ bool Arch::route()
     } else {
         log_error("FPGA interchange architecture does not support router '%s'\n", router.c_str());
     }
+
+    if (result) {
+        // Fixup LUT vcc pins.
+        IdString vcc_net_name(chip_info->constants->vcc_net_name);
```

Comment:
I think it'd be neater to have this in another function like `Arch::fixup_lut_vcc` to make `route` a bit simpler

Reply:
I agree with that, I'll refactor it.

Reply:
Done